app/views/docs/command-line-deployment.phtml

```diff
@@ -47,7 +49,9 @@ The Apprite CLI allows you to create and deploy databases, collections, buckets,
 
 <h2><a href="/docs/command-line-deployment#deployCollections" id="deployCollections">Deploying Databases and Collections</a></h2>
 
-<p>The Appwrite CLI also helps you migrate your project's databases and collections from a development server to a production server. You can deploy all the databases and collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file using:</p>
+<p>The Appwrite CLI also helps you migrate your project's databases and collections from a development server to a production server.</p>
+
+<p>You can deploy all the databases and collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file by running this command in the folder holding your <code>appwrite.json</code> file.</p>
```

Comment:
we should add a note that, currently, the CLI will remove and re-create attributes and indexes for existing collections (which would wipe the data in documents).

Reply:
Oh. my. You should really not suggest they use CLI at all then